guide/src/module.md

```diff
@@ -48,53 +53,110 @@ To import the module, either:
 
 ## Documentation
 
-The [Rust doc comments](https://doc.rust-lang.org/stable/book/first-edition/comments.html) of the module
+The [Rust doc comments](https://doc.rust-lang.org/stable/book/ch03-04-comments.html) of the module
 initialization function will be applied automatically as the Python docstring of your module.
 
+For example, building off of the above code, this will print `This module is implemented in Rust.`:
+
 ```python
-import rust2py
+import my_extension
 
-print(rust2py.__doc__)
+print(my_extension.__doc__)
 ```
 
-Which means that the above Python code will print `This module is implemented in Rust.`.
+## Organizing your module registration code
+
+As your Python module grows, it can be helpful to split your Rust code into several Rust modules to
+keep your code readable.
+
+You can pass references to the `PyModule` so that each 
+module registers its own FFI code, which can give your project better 
```

Comment:
Is FFI a safe word to use here? Generally acronyms should be avoided in docs.

Reply:
Yes I think FFI is fairly prevalent in PyO3's guide. TBH we could add a glossary if we really were worried, to include other terms like GIL which we also use a lot.